pyaerocom/ungriddeddata.py

```diff
@@ -1966,6 +1985,8 @@ def _new_from_meta_blocks(self, meta_indices, totnum_new):
             new.metadata[meta_idx_new] = meta
             new.meta_idx[meta_idx_new] = {}
             for var in meta["var_info"]:
+                if var == "altitude":
```

Comment:
This should test against `ALLOWED_VERT_CORD_TYPES`, not just `"altitude"`

Reply:
Good call